lab_5_scrapper/scrapper.py

```diff
@@ -2,9 +2,49 @@
 Crawler implementation.
 """
 # pylint: disable=too-many-arguments, too-many-instance-attributes, unused-import, undefined-variable
+import datetime
+import json
 import pathlib
+import random
+import re
+import time
 from typing import Pattern, Union
 
+import requests
+from bs4 import BeautifulSoup
+
+from core_utils import constants
+from core_utils.article.article import Article
+from core_utils.article.io import to_meta, to_raw
+from core_utils.config_dto import ConfigDTO
+
```

Comment:
See import style checks log output:

lab_5_scrapper/scrapper.py:21:1: I003 isort expected 1 blank line in imports, found 0

-> add one blank line here